src/app/components/search/search-input/search-input.component.ts

```diff
@@ -93,13 +96,22 @@ export class SearchInputComponent implements OnInit, OnDestroy {
   @ViewChild('searchInputControl')
   searchInputControl: SearchInputControlComponent;
 
+  @ViewChild(MatMenuTrigger)
+  trigger: MatMenuTrigger;
+
   constructor(
     private queryBuilder: SearchQueryBuilderService,
     private queryLibrariesBuilder: SearchLibrariesQueryBuilderService,
+    private config: AppConfigService,
     private content: ContentManagementService,
     private router: Router,
     private store: Store<AppStore>
-  ) {}
+  ) {
+    this.searchOnChange = this.config.get<boolean>(
+      'search.triggeredOnChange',
```

Comment:
Changing name to see that it's not the core ADF property yet.
```suggestion
      'search.aca:triggeredOnChange',
```